### DISCUSSION TITLE
SPECFEM Developers Workshop at SSA24 — April 30, 2024

### DISCUSSION THREAD

**Discussion by bch0w** [SPECFEM/specfem3d#1664] [General]
We are hosting a hybrid 1-day SPECFEM Developers Workshop on Tuesday April 30, 2024 prior to the 2024 SSA annual meeting in Anchorage, Alaska (see #1663 for SSA announcement)! 

Come join us that afternoon for a few hours of SPECFEM-related discussions, followed by a group dinner in the evening after the SSA plenary talk. We have a great lineup of talks and well as time for group discussion on the current state and future direction of SPECFEM.

### Important Information
**What:** 2024 SPECFEM Developers Workshop
**When:** Tuesday April 30, 2024 15:30–18:00 AKDT (workshop); 19:30–21:00 AKDT (dinner)
**Where:** [William A. Egan Civic & Convention Center](https://maps.app.goo.gl/7rxzK9PqvgQerqVc7) and Online (please [register](https://www.eventbrite.com/e/2024-specfem-developers-meeting-registration-815894521477) to get Zoom link)
**Why:** To share and discuss SPECFEM and related research

### Important Links
Website: https://geodynamics.org/events/details/343
Registration: https://www.eventbrite.com/e/2024-specfem-developers-meeting-registration-815894521477

Looking forward to seeing you in Alaska!
SPECFEM Developers & Maintainers

**Comment by ljhwang** [comment]
Registration is open for the SPECFEM Workshop.  Please see:  [website](https://geodynamics.org/events/details/343)

**Comment by carltape** [comment]
Hi folks, please register for the SSA specfem workshop if you will be there in Anchorage on April 30th. You contributions and participation are needed! Dinner included! And we also would like to know who plans to participate remotely.
Registration is open for the SPECFEM Workshop. Please see: [website](https://geodynamics.org/events/details/343)

**Comment by bch0w** [comment]
Last call for workshop registration! **In-person registration will be closing April 24th** (tomorrow)! 
See link for registration: https://geodynamics.org/events/details/343

We have a great schedule lined up with a number of talks and open discussion topics (see attached PDF below)
Virtual registration will remain open but please sign up asap!

[Click to download schedule](https://github.com/SPECFEM/specfem3d/files/15080273/specfem_dev_workshop24_schedule.pdf)

**Comment by ICNSK** [comment]
Hi, Any plans to record workshop presentations and share them with general users who will not be able to attend ? Similar to https://specfem.org/training

**Reply by bch0w** [reply]
Hi @ICNSK, yes we should be recording, although I'm not sure how widely we will distribute, that will be up to the speakers/audience.

I would suggest registering as a virtual participant, as I can envision distributing the recordings to the list of participants at a minimum. Thanks!
